spec/view_models/admin/user_auction_view_model_spec.rb

```diff
@@ -24,7 +24,7 @@
   describe '#user_bid_count' do
     it 'should return the total number of bids the user has made' do
       user2 = create(:user)
-      auction = create(:auction, :with_bidders, bidder_ids: [user.id, user2.id, user.id, user2.id, user.id])
+      auction = create(:auction, bidders: [user, user2, user, user2, user])
```

Comment:
One thing I would love is the ability to pass it something like `nil` or `:new` and just let the factory create the other bidders I don't care about, but I might be overthinking